### PR TITLE
No -result refactored

### PR DESCRIPTION
**Pull request by HugoWith**
## Objective

What:
👉 Refactoring no result
👉 Create a Route component
👉 Getting query on refresh and if no results direclty land on no result page
👉 Fix flickering on refresh (infinite loop)

## Type

- [x] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [ ] Style Tweaks
- [x] Code Refactoring
- [ ] Documentation
- [ ] Tests

## Tested

locally tested

## Translation

- [ ] Have you translated all your Paragraphs, ButtonsTitle,... with i18n functionality?

## Documented

- [x] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

### CODE REVIEW

File: src/pages/searchResultsPage/SearchResultsPage.jsx [hunk 1]
```diff
@@ -166,7 +158,7 @@ const SearchResultsPage = () => {
 
   return (
     <>
-      {!showResults && <NoResults />}
+      {/* {!showResults && <NoResults />} */}
```
Comment (matthew1809) [inline src/pages/searchResultsPage/SearchResultsPage.jsx:161]: Why not remove?

File: src/pages/searchResultsPage/SearchResultsPage.jsx [hunk 2]
```diff
@@ -144,7 +136,7 @@ const SearchResultsPage = () => {
   }, [navigationState])
 
   let configureProps = {
-    hitsPerPage: hitsPerPageNotInjected,
+    // hitsPerPage: hitsPerPageNotInjected,
```
Comment (matthew1809) [inline src/pages/searchResultsPage/SearchResultsPage.jsx:139]: Why not remove?

File: src/pages/searchResultsPage/SearchResultsPage.jsx [hunk 3]
```diff
@@ -99,7 +91,7 @@ const SearchResultsPage = () => {
   const { injectedContentIndex } = useRecoilValue(indexNames)
 
   // Define Stat Const
-  const { hitsPerPageNotInjected } = hitsPerPage
+  // const { hitsPerPageNotInjected } = hitsPerPage
```
Comment (matthew1809) [inline src/pages/searchResultsPage/SearchResultsPage.jsx:94]: Why not remove?

**Review by matthew1809 — APPROVED** [review]

**Comment by HugoWith** [comment]
Thanks @matthew1809 It's all removed 🙏

**Pull request merged**